DependencyInjection/Configuration.php

```diff
@@ -573,15 +573,18 @@ private function createCachePluginNode()
                         ->end()
                     ->end()
                 ->end()
-                ->enumNode('respect_cache_headers')
+                ->scalarNode('respect_cache_headers')
                     ->info('Whether we should care about cache headers or not [DEPRECATED]')
-                    ->values([null, true, false])
                     ->beforeNormalization()
                         ->always(function ($v) {
                             @trigger_error('The option "respect_cache_headers" is deprecated since version 1.3 and will be removed in 2.0. Use "respect_response_cache_directives" instead.', E_USER_DEPRECATED);
 
                             return $v;
                         })
+                        ->end()
```

Comment:
the end call should be unindented to make the tree structure visual. This is the end of `->beforeNormalization()`